docs/connect/index.md

```diff
@@ -38,34 +45,41 @@ applications and drivers may obtain connection properties in different formats.
 :padding: 0
 
 :::{grid-item}
-:columns: 5
+:columns: 4
 :margin: 0
 :padding: 0
 
 **Connection properties**
 
 :Host: `<clustername>`.cratedb.net
-:Port: 5432 (PostgreSQL) or 4200 (HTTP)
+:Port: 5432 (PostgreSQL) or<br>4200 (HTTP)
 :User: `<username>`
 :Pass: `<password>`
 
 :::
 
 :::{grid-item}
-:columns: 7
+:columns: 8
 :margin: 0
 :padding: 0
 
 **Connection-string examples**
 
-A native PostgreSQL connection string.
-`postgresql://<username>@<clustername>.cratedb.net/crate`
+A native PostgreSQL connection string, e.g. for `psql`.
+<br>
+`postgresql://<username>:<password>@<clustername>.cratedb.net:5432/`
```

Comment:
```suggestion
`postgresql://<username>:<password>@<clustername>.cratedb.net:5432/crate`
```

I think some applications might require a database name or fall back to e.g. `public`

Reply:
Do you think this is also applicable for the JDBC variant? I can find two spots here, one adds the `/crate` suffix, the others don't.

- https://github.com/crate/cratedb-examples/blob/main/application/apache-kafka-flink-streaming/flink_consumer.py#L22
- https://github.com/crate/cratedb-flink-jobs/blob/main/src/main/java/io/crate/streaming/TaxiRidesStreamingJob.java#L37-L42
- https://github.com/crate/pre-sales/blob/master/Flink/LongshoremanFlink/src/main/java/io/crate/longshoreman/MongoToCrateJob.java#L36-L38

Reply:
Do you have any ideas if `/crate` (or even `/doc`?) MUST be used as the `path` fragment of a connection URI when connecting with vanilla pgJDBC and/or CrateDB JDBC, @surister or @kneth?

What do you suggest to display on the canonical documentation spot?

Reply:
The canonical example exercising both drivers [cratedb-examples://by-language/java-jdbc](https://github.com/crate/cratedb-examples/tree/main/by-language/java-jdbc) doesn't use any path suffix, so I guess it will be good enough to also omit it on the documentation?

-- https://github.com/crate/cratedb-examples/blob/main/by-language/java-jdbc/Makefile

Reply:
Nothing is "required" other then `jdbc:postgresql:`

https://github.com/pgjdbc/pgjdbc?tab=readme-ov-file#building-the-connection-url

Reply:
On another note, if I can find the space, I may add another (foot)note about this detail you educated me about. There, information about how to address a _database schema_, might be sensible. In SQLAlchemy, it is by using the `?schema=` URL parameter, but I don't actually know whether and how this works for crash, or JDBC.

Reply:
That's two more samples by @kneth:

- ["jdbc:crate://localhost:5432/", connectionProps](https://github.com/kneth/CrateJDBCTestbed/blob/3e86de5/app/src/main/java/cratejdbcexperiments/App.java#L18)
- ["jdbc:postgresql://localhost:5432/doc?user=crate"](https://github.com/kneth/CrateJDBCTestbed/blob/3e86de5/app/src/main/java/cratejdbcexperiments/App.java#L33)

Reply:
That's two more connection string samples by @kneth:

- ["jdbc:crate://localhost:5432/", connectionProps](https://github.com/kneth/CrateJDBCTestbed/blob/3e86de5/app/src/main/java/cratejdbcexperiments/App.java#L18)
- ["jdbc:postgresql://localhost:5432/doc?user=crate"](https://github.com/kneth/CrateJDBCTestbed/blob/3e86de5/app/src/main/java/cratejdbcexperiments/App.java#L33)

Reply:
It looks like the `/path` suffix is currently also interpreted as the addressed _schema_, and seems to work reasonably well when used with `psql`?

By default, CrateDB has a built-in catalog `crate`, and a default schema `doc`, right? That's why addressing an unknown schema `foo` fails with a corresponding error message.
```
$ psql "postgresql://crate@localhost:5432/foo" -c "\d"
ERROR:  Schema 'foo' unknown
```

Now, when populating existing and non-existing schemas with tables, like:
```shell
psql "postgresql://crate@localhost:5432/" -c "CREATE TABLE crate.crate (id INT);"
psql "postgresql://crate@localhost:5432/" -c "CREATE TABLE doc.doc (id INT);"
psql "postgresql://crate@localhost:5432/" -c "CREATE TABLE foo.foo (id INT);"
```
... **all of them** can be addressed well, by means of the `/path` suffix to the connection string URI.
```shell
psql "postgresql://crate@localhost:5432/crate" -c "\d"
psql "postgresql://crate@localhost:5432/doc" -c "\d"
psql "postgresql://crate@localhost:5432/foo" -c "\d"
```

Reply:
Coming from all those observations, I guess this one should be the canonical connection string to advertise, for both `psql` and JDBC use, effectively using `/doc` instead of `/crate`, on both of them. Let me know if you have any objections, because I may still got something wrong. Thanks!

```
postgresql://crate@localhost:5432/doc
```

Reply:
> The canonical example exercising both drivers [cratedb-examples://by-language/java-jdbc](https://github.com/crate/cratedb-examples/tree/main/by-language/java-jdbc) doesn't use any path suffix, so I guess it will be good enough to also omit it on the documentation, for the JDBC/Flink cases?
>
> ```shell
> mvn exec:java -Dexec.args="--dburl 'jdbc:postgresql://localhost:5432/'"
> mvn exec:java -Dexec.args="--dburl 'jdbc:crate://localhost:5432/'"
> ```

I've just exercised it also with the JDBC rig, and the behaviour is the same: The `/path` suffix selects the target _schema_ in CrateDB. When omitted, the vanilla pgJDBC driver will use the username as default, like @proddata mentioned about DBeaver, while the CrateDB JDBC driver will use `doc` by default.

Taking this into consideration, I think the safest way is to **always** tuck `/doc` onto the advertised connection URIs, in order to minimize confusion.

Reply:
Too much augmented by a layout change to sensibly decipher anything from that, a40687633b now uses the `doc` schema everywhere.

Reply:
Going to resolve this. Let me know if you have anything to add.